.github/workflows/deploy-dev.yml

```diff
@@ -65,3 +65,9 @@ jobs:
           make plan
           make deploy-infra
           make deploy-app
+
```

Comment:
Do we need to wait a bit to ensure deployment ?

Reply:
I believe our TF process waits for cloudfront to be fully updated before it finishes